src/_guides/language/language-tour.md

```diff
@@ -4174,11 +4174,13 @@ For more information, see the following:
   including [Isolate.spawn()][] and
   [TransferableTypedData][]
 * [Background parsing][background json] cookbook on the Flutter site
+* [Example of using isolates][]
```

Comment:
nit: Consider making it clear that this is a full-blown sample app, like "Example app demonstrating isolates" or "Isolate sample app" or something.